src/routes/team/[team]/[env]/app/[app]/+page.svelte

```diff
@@ -129,7 +129,7 @@
 	</div>
 	<Confirm bind:open={restart} on:confirm={submit}>
 		<h3 slot="header">Restart {app}</h3>
-		This will restart all instances of <strong> {app}</strong> in <strong>{env}</strong>.
+		This will restart all instances of<strong> {app}</strong> in <strong>{env}</strong>.
```

Comment:
Ekstrem nitpick, feel free to ignore :D 
```suggestion
		This will restart all instances of <strong>{app}</strong> in <strong>{env}</strong>.
```